src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/mobile/update_turn_list_request_test.cc

```diff
@@ -241,6 +241,66 @@ TEST_F(UpdateTurnListRequestTest, Run_ValidTurnList_SUCCESS) {
                        .asString());
 }
 
+TEST_F(UpdateTurnListRequestTest, Run_ValidTurnList_WARNINGS) {
+  const std::string kNavigationText = "valid_navigation_text";
+
+  (*command_msg_)[am::strings::msg_params][am::strings::turn_list][0]
+                 [am::strings::navigation_text] = kNavigationText;
+  (*command_msg_)[am::strings::msg_params][am::strings::turn_list][0]
+                 [am::strings::turn_icon][am::strings::value] =
+                     "valid_turn_icon";
+  (*command_msg_)[am::strings::msg_params][am::strings::soft_buttons] = 0;
+
+  MockAppPtr mock_app(CreateMockApp());
+  EXPECT_CALL(app_mngr_, application(kConnectionKey))
+      .WillOnce(Return(mock_app));
+
+  EXPECT_CALL(mock_message_helper_,
+              ProcessSoftButtons((*command_msg_)[am::strings::msg_params],
+                                 Eq(mock_app),
+                                 Ref(mock_policy_handler_),
+                                 Ref(app_mngr_)))
+      .WillOnce(Return(mobile_result::SUCCESS));
+
+  EXPECT_CALL(
+      mock_message_helper_,
+      VerifyImage(
+          (*command_msg_)[am::strings::msg_params][am::strings::turn_list][0]
+                         [am::strings::turn_icon],
+          Eq(mock_app),
+          Ref(app_mngr_))).WillOnce(Return(mobile_result::WARNINGS));
+
+  EXPECT_CALL(mock_message_helper_,
+              SubscribeApplicationToSoftButton(_, _, kFunctionId));
+
+  MessageSharedPtr result_msg(CatchHMICommandResult(CallRun(*command_)));
+  ASSERT_TRUE(result_msg);
```

Comment:
`(bool)result_msg`

Reply:
@jacobkeeler fixed in https://github.com/smartdevicelink/sdl_core/pull/2276/commits/51e40bf42f03ed368ac51123b77444313a2fe84d